release.json

```diff
@@ -8,6 +8,14 @@
 		"JMXFETCH_VERSION": "0.26.1",
 		"JMXFETCH_HASH": "e25fdb6173ea357d6c161228bba7a4c0b60151c89e1bb094f17850d475e4616e"
 	},
+	"6.11.0-rc.1": {
+		"INTEGRATIONS_CORE_VERSION": "6.11.0-rc.1",
+		"PROCESS_AGENT_VERSION": "6.11.0-rc.1",
```

Comment:
Process is missing a 6.11.0-rc.1 tag https://github.com/DataDog/datadog-process-agent/releases